### ISSUE TITLE
as of this evening, stable doesn't install correctly on Ubuntu 16 LTS

### ISSUE THREAD

**Issue by john3641** [tethysplatform/tethys#436]
I understand that the tethys platform must be especially difficult to maintain and continually improve because it interacts with so much other software.  And there's no way I could even hope to do the project I'm determined to complete  without tethys.  But my impression was that the stable version was the version that has already proven itself to be stable, and had no way of knowing that it could be changed at all without notice, let alone changed to one that generates errors from the get-go, on an operating system that is widely used. (See installation errors pasted below.  Errors appeared after much of tethys seemed to install successfully.) Could the previous version that is known to be stable be made available.  I dread facing Monday morning with this unresolved.  Also, is there a way that I can know whenever the last update is made?  Maybe always have a comment line at the top of the install_tethys.sh script that has the date of last update, and in the tutorial state something like "This procedure installs the stable version of tethys.  Although it is made available only after thorough testing, errors in installation are possible.  If you experience such errors,, please note the date of the last update in the comment line at the top of the script..."  Forgive me if something like this is already done, and I've missed it.  I'm feeling very frustrated right now, but I am thankful for tethys.

...
running bootstrap script ... ok
performing post-bootstrap initialization ... 2019-06-21 17:09:59.679 CDT [7252] FATAL:  could not load library "/home/john/tethys/miniconda/envs/tethys/lib/ascii_and_mic.so": /home/john/tethys/miniconda/envs/tethys/lib/ascii_and_mic.so: undefined symbol: check_encoding_conversion_args
2019-06-21 17:09:59.679 CDT [7252] STATEMENT:  -- SQL_ASCII --> MULE_INTERNAL
    CREATE OR REPLACE FUNCTION ascii_to_mic (INTEGER, INTEGER, CSTRING, INTERNAL, INTEGER) RETURNS VOID AS '$libdir/ascii_and_mic', 'ascii_to_mic' LANGUAGE C STRICT PARALLEL SAFE;
    COMMENT ON FUNCTION ascii_to_mic(INTEGER, INTEGER, CSTRING, INTERNAL, INTEGER) IS 'internal conversion function for SQL_ASCII to MULE_INTERNAL';
    CREATE DEFAULT CONVERSION pg_catalog.ascii_to_mic FOR 'SQL_ASCII' TO 'MULE_INTERNAL' FROM ascii_to_mic;
    COMMENT ON CONVERSION pg_catalog.ascii_to_mic IS 'conversion for SQL_ASCII to MULE_INTERNAL';
    
child process exited with exit code 1
initdb: removing data directory "/home/john/tethys/psql/data"
john@john-VirtualBox:~$ . ~/.bashrc
john@john-VirtualBox:~$ t
(tethys) john@john-VirtualBox:~$ tstart
No command 'tstart' found, did you mean:
 Command 'rstart' from package 'x11-session-utils' (main)
 Command 'kstart' from package 'kde-runtime' (universe)
 Command 'start' from package 'upstart' (main)
 Command 'qstart' from package 'torque-client' (universe)
 Command 'qstart' from package 'torque-client-x11' (universe)
tstart: command not found
(tethys) john@john-VirtualBox:~$ tms
No command 'tms' found, did you mean:

**Comment by biplovbhandari** [comment]
I get the same error with ubuntu 18 LTS

**Comment by swainn** [comment]
@john3641 I'm sorry you are having difficulty with installing Tethys. I have confirmed the issue. My guess is that a new version of the postgresql conda dependency is broken or incompatible. I'm working on fix right now.

**Comment by swainn** [comment]
The postgresql package was updated on Friday:

postgresql 11.4 hbeee2d4_0
--------------------------
file name   : postgresql-11.4-hbeee2d4_0.tar.bz2
name        : postgresql
version     : 11.4
build       : hbeee2d4_0
build number: 0
size        : 4.8 MB
subdir      : linux-64
url         : https://conda.anaconda.org/conda-forge/linux-64/postgresql-11.4-hbeee2d4_0.tar.bz2
md5         : 793c4017174cc41f34ae85e76ec64dc0
timestamp   : 2019-06-21 16:28:16 UTC
dependencies: 
  - krb5 >=1.16.3,<1.17.0a0
  - libgcc-ng >=7.3.0
  - libpq 11.4 h4e4e079_0
  - libxml2 >=2.9.9,<2.10.0a0
  - openssl >=1.1.1a,<1.1.2a
  - readline >=7.0,<8.0a0
  - tzcode
  - zlib >=1.2.11,<1.3.0a0

**Comment by swainn** [comment]
@john3641 @biplovbhandari I've submitted a pull request to address this issue (see: #437). Waiting for review to merge it.

**Comment by sdc50** [comment]
PR #437 was just merged. Please try installation again and let us know if you encounter any other errors. Thanks for reporting about this issue.

**Comment by swainn** [comment]
Please delete the `tethys` directory and the old `install_tethys.sh` before you try again.

**Comment by john3641** [comment]
OK, it works now.  Thanks!

**Comment by biplovbhandari** [comment]
works with 18 and 19 as well. Thanks!
